src/pymoca/ast.py

```diff
@@ -756,6 +833,76 @@ def _find_constant_symbol(self, component_ref: ComponentRef, search_parent=True)
     def find_constant_symbol(self, component_ref: ComponentRef) -> Symbol:
         return self._find_constant_symbol(component_ref)
 
+    def _find_symbol(self, component_ref: ComponentRef, search_parent=True, in_original_scope=False) -> Symbol:
+        if component_ref.child:
+            # Try classes first, and constant symbols second
+            t = component_ref.to_tuple()
+
+            try:
+                node = self._find_class(ComponentRef(name=t[0]), search_parent)
```

Comment:
Probably faster to make this a `copy=False` lookup. That's also strictly speaking more correct.